setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -123,4 +124,28 @@ public function validate(InputInterface $input)
 
         return $errors;
     }
+
+    /**
+     * Get admin user data
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function setMissingValues(InputInterface &$input, OutputInterface $output)
+    {
+        $dialog = $this->getHelper('dialog');
+        $keys = [
+            AdminAccount::KEY_FIRST_NAME,
+            AdminAccount::KEY_LAST_NAME,
+            AdminAccount::KEY_USER,
+            AdminAccount::KEY_EMAIL,
+            AdminAccount::KEY_PASSWORD,
+        ];
+        foreach ($keys as $key) {
+            if (!$input->getOption($key)) {
+                $input->promptForOption($dialog, $output, $key, $key === AdminAccount::KEY_PASSWORD);
```

Comment:
Please check that input actually implements the interface which has promptForOption method